includes/admin/class-amp-post-meta-box.php

```diff
@@ -213,6 +213,26 @@ public function enqueue_block_assets() {
 			return;
 		}
 
+		$gb_supported = defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '5.3.0', '>=' );
```

Comment:
Why 5.3.0 specifically?

Reply:
It was the first stable release that worked without any errors. I guess I could bump it to 5.4.0 since that was the version bundled with WP 5.2.